### PR TITLE
chore: cleanup uniswap deps

### PR DESCRIPTION
**Pull request by julien51**
# Description

In order to remove a few deps, I am cleaning up some code.

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: packages/hardhat-helpers/src/uniswap.js [hunk 1]
```diff
@@ -1,104 +1,18 @@
 const { ethers } = require('ethers')
 
 const { AllowanceTransfer } = require('@uniswap/permit2-sdk')
-const {
-  Token,
-  CurrencyAmount,
-  TradeType,
-  Percent,
-} = require('@uniswap/sdk-core')
+const { Token } = require('@uniswap/sdk-core')
```
Comment (julien51) [inline packages/hardhat-helpers/src/uniswap.js:4]: I would love to get rid of this completely. It looks like this (and `nativeOnChain`) are used for `getUniswapTokens` but I think we may not "need" this format? We could easily use the tokens from our own network configs?

What do you think @clemsos ?

Comment (clemsos) [reply packages/hardhat-helpers/src/uniswap.js:4]: yes we dont need this anymore. The `getUniswapTokens` is still used in some tests but we could use our package instead

Comment (julien51) [reply packages/hardhat-helpers/src/uniswap.js:4]: ok I will try to replace that then!

**Review by clemsos — APPROVED** [review]
deps cleanup !

**Pull request merged**